email-provider/src/main/java/life/qbic/infrastructure/email/Attachment.java

```diff
@@ -0,0 +1,10 @@
+package life.qbic.infrastructure.email;
+
+/**
+ * <b>Email Attachment</b>
+ *
+ * @since 1.0.0
+ */
+public record Attachment(String name, String content) {
```

Comment:
There are multiple Attachment Classes?

Reply:
one is a left over, probably have overseen it

Reply:
ah no, look at the modules! The concept is treated uniquely for each application core / domain. Identity vs Project Management. A little price to pay for duplicate code vs clean separation of application cores.